stories/Colors/Colors.story.jsx

```diff
@@ -1,136 +1,34 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { pick } from 'lodash';
-import colors from '../../src/theme/Colors';
-import { H2 } from '../../src/components/Typography/Typography';
-import { Box } from '../../src/components/Layout';
-import { hashColors, processColor } from '../../src/utils/colors';
-import ColorSpecimen, { GradientSpecimen } from './Colors';
+import Swatch from './Colors';
+import '../../src/css/tailwind.css';
 
-const uikitStories = storiesOf('Colors', module);
+const stories = storiesOf('Colors', module);
 
-const brandColors = hashColors(pick(colors, ['primary', 'secondary', 'tertiary', 'white']));
-
-uikitStories.add('Brand', () => (
-  <div
-    style={{
-      display: 'flex',
-      flexWrap: 'wrap',
-      flexDirection: 'row',
-    }}
-  >
-    {Object.keys(brandColors).map(name => (
-      <ColorSpecimen key={name} color={brandColors[name].hex} name={name} />
-    ))}
+const brand = ['primary', 'secondary', 'tertiary'];
+const brandSwatches = brand.map(color => <Swatch color={color} />);
```

Comment:
this is where we can leverage the original js theme for both tailwind configs and storbook. The theme will be the single point of truth